ocs_ci/deployment/deployment.py

```diff
@@ -111,6 +111,8 @@
 )
 from ocs_ci.ocs.ui.helpers_ui import ui_deployment_conditions
 from ocs_ci.utility.utils import get_az_count
+from ocs_ci.utility.ibmcloud import run_ibmcloud_cmd
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
```

Comment:
This is not used and the flake8 is failing because of that.

Reply:
Changed and pushed the changes